packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -768,6 +775,16 @@ export default [
                       value={state.concurrency}
                     />
                   </FormGroup>
+                  <FormGroup>
+                    <label>
+                      <strong>{_('offlineSnapshot')}</strong>
+                    </label>{' '}
+                    <input
```

Comment:
Either put the `input` inside the `label` or use a (generated) `id` to link them.